java/client/src/main/java/glide/api/commands/StreamBaseCommands.java

```diff
@@ -105,4 +105,29 @@ public interface StreamBaseCommands {
      * }</pre>
      */
     CompletableFuture<Long> xdel(String key, String[] ids);
+
+    /**
+     * Returns stream entries matching a given range of IDs.
+     *
+     * @param key The key of the stream.
+     * @param start Starting ID to search. Use <code>"-"</code> to start with the minimum possible ID.
+     *     Include a <code>"("</code> prior to the ID to do an exclusive search.
+     * @param start End ID to search, or <code>"+"</code> to end with the maximum possible ID. Include
+     *     a <code>"("</code> prior to the ID to do an exclusive search.
```

Comment:
I think `start` and `end` should be an option

Reply:
I was thinking of doing that... which the options being an ENUM with +/-,  with an optional exclusive search. 
Is that what you're thinking...?

Reply:
Yes, we have similar thing for `zrange` commands made by San the Great.